src/App.js

```diff
@@ -6,16 +6,20 @@ import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 import TodoList from "./pages/TodoList";
 import IncompleteTasks from "./pages/IncompleteTasks";
 import CompletedTasks from "./pages/CompletedTasks";
+import AddTaskForm from "./../src/pages/AddTasksForm";
 
 function App() {
 	return (
 		<div className="App">
 			<Router>
 				<Switch>
-					<Route path="/incomplete">
+					<Route path="/add-tasks">
```

Comment:
This spacing seems really big. Good thing - it's consistent across all of the files, but still makes it harder to read.